### DISCUSSION TITLE
How to remove battery status and card memory icons while streaming in Twitch through OBS

### DISCUSSION THREAD

**Discussion by TheBanyTrip** [gopro/labs#529] [Q&A] [answered]
Hey, anyone of you guys know how to remove the battery status and the card memory icons? I've installed GoPro Labs already and scanned the QR overlay "Add an Overlay to Video" (no time, no date) but when I connect the HDMI to the GoPro, I still can see the icons in my OBS screen. Thanks a lot for your help.
<img width="1488" alt="Captura de pantalla 2023-09-13 a las 12 00 05" src="https://github.com/gopro/labs/assets/135733916/c2841063-67b3-4841-8d13-4644cc27ec80">

**Comment by dnewman-gpsw** [comment, marked as answer]
!MHDMI=1

This clears the overlays.

**Reply by TheBanyTrip** [reply]
Thanks for answering and your quick reply. I've already tried  but I need something to remove the SD card and the battery status while I'm blogging through OBS. Thanks again.

<img width="948" alt="Captura de pantalla 2023-09-13 a las 17 34 14" src="https://github.com/gopro/labs/assets/135733916/4c1eaa1f-b369-4fb6-8822-dfdadd62ad5f">

**Reply by dnewman-gpsw** [reply]
This is that suggestion does, it clears HDMI of overlays.  Sorry typo !MHDMI=1

From https://gopro.github.io/labs/control/extensions/  

HDMI=0,1 or 2 - Media Mod users can change the output default from Gallery (0) to clean monitoring with no overlays (1), or monitoring live video with overlays (2).  Supported on HERO12/11/10/9.

**Reply by TheBanyTrip** [reply]
I was getting crazy. It was the comand ended in 1 to clean the screen. THANKS FOR YOUR KINDLY HELP! 🙏

<img width="667" alt="Captura de pantalla 2023-09-13 a las 18 55 46" src="https://github.com/gopro/labs/assets/135733916/e25c4485-81dc-45ea-8ac6-65d468dcbb7f">
<img width="691" alt="Captura de pantalla 2023-09-13 a las 18 57 48" src="https://github.com/gopro/labs/assets/135733916/6533ca14-7a76-43f4-8aaa-1c0c0fa87cb4">
